app/controllers/User.scala

```diff
@@ -31,6 +31,15 @@ object User extends LilaController {
     }
   }
 
+  def studyTv(username: String) = Open { implicit ctx =>
+    OptionFuResult (UserRepo named username) { user =>
+      lila.relation.Env.current.onlineStudying.get(user.id) match {
+        case None => notFound
```

Comment:
Doesn't that defeat the point of having the redirect to the study you're currently working on?